### PR TITLE
Add contact page.

### PR DESCRIPTION
**Pull request by IshaDakota**

### CODE REVIEW

**Comment by IshaDakota** [comment]
fireshot capture 19 - contact gizra - http___0 0 0 0_9000_contact_

**Comment by IshaDakota** [comment]
#384 
@RachelBaram please review language
@amitaibu please review code.

File: app/contact/Untitled.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
```
Comment (amitaibu) [inline app/contact/Untitled.html]: this one is probably wrong.

Comment (amitaibu) [inline app/contact/Untitled.html]: I'll remove it

File: app/contact/Untitled.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+											</div>
+											<div class="form-group">
+												<label for="project">Tell us a little about your project:</label>
+												<textarea name="project" class="form-control" rows="3"></textarea>
+											</div>
+										  <button type="submit" class="btn btn-default">Send</button>
+										  <input type="hidden" name="_next" value="//gizra.com/contact/thank-you" />
+										</form>
+										<div class="">
+                      <p><span>Or just email us at </span><a class="email" href="mailto:usoffice@gizra.com">usoffice@gizra.com</a></p>
```
Comment (amitaibu) [inline app/contact/Untitled.html]: going to change it to `info@gizra.com`

**Comment by amitaibu** [comment]
I've added some bottom margin

contact_gizra

**Comment by amitaibu** [comment]
Merged, deployed and email confirmed.

Thanks!

**Comment by IshaDakota** [comment]
@amitaibu didn't see that you confirmed... you'll get a test email from me. ignore obv.

**Pull request merged**